### ISSUE TITLE
[Feature Section]: Expand the availability of CTA types in feature card sections.

### ISSUE THREAD

**Issue by m4olivei** [carbon-design-system/carbon-for-ibm-dotcom#11159]
### The problem

The Feature card section only has support for two CTA types in Storybook:

* Local (local)
* External (external)

Under the hood however, the relevant component, `<dds-feature-section-card-link>` extends `<dds-card-cta>`, which supports a bunch of CTA types:

* Local (local)
* Jump (jump)
* External (external)
* New Tab (new tab)
* Download (download)
* Video (video)
* PDF (pdf)
* Blog (blog)
* Email (email)
* Schedule (schedule)
* Chat (chat)
* Call (call)

When used, most of these work as desired, aside from "Video (video)", which has some undesireable results:

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/191049/9d4c7933-081c-4d63-ada9-186af1a5ec0f)

### The solution

Update the Feature section story with the following:

* Allow the user to select from all `<dds-card-cta>` supported CTA types
* Wrap the Feature section story in `<dds-video-cta-container>` to support the Video CTA type specifically
* Fix the double heading bug in the screenshot above

### Application/website

https://ibm.com

### Business priority

High Priority = pressing release

### What time frame would this ideally be needed by (if applicable)

Prior to the next release, as soon as possible. PR w/ solution is incoming.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by ariellalgilmore** [comment]
@m4olivei the v2 feature section currently shows all the cta options: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/v2/web-components/?path=/story/components-feature-section--default

**Comment by ariellalgilmore** [comment]
ahh i see yes that might still be an issue in v2 or might just be a storybook issue not exactly sure. 

Maybe @kennylam @RichKummer can help answer the second question on how to get this into v1?

**Comment by andy-blum** [comment]
I think to get this into V1 we need to target the `v1` branch. Not sure how fixing this in _both_ locations needs to happen. I know in other open source projects the fix gets put in the _current_ main dev branch and is cherry picked into older versions.

**Comment by m4olivei** [comment]
I've switched the base branch to `v1`. That's our immediate need for the fix anyway. Of course, happy to open a PR against `main` [once `v2` lands](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/11161).

**Comment by m4olivei** [comment]
This was merged into v1 in https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/11160 (now released in 1.54.0) and already complete for v2. Closing.
